export/export_pipeline/compute_collision_representation.py

```diff
@@ -0,0 +1,427 @@
+                json.dump(components_info, f, indent=4)
+
+            # group the leaves by group_id
+            # TODO(samarth) group name 
+            geom_groups = defaultdict(list)
+            for geom in geoms:
+                geom_groups[geom[-1]].append(geom[:-1])
+            
+            for geom_group in geom_groups.values():
+                geom_dir, geom, decompose_method = geom_group[0]
+                leaf_name = geom.name.split('.')[0]
+                
+                # merge the geoms
+                for l in geom_group[1:]:
+                    l[1].select_set(True)
+                    shutil.rmtree(l[0])
+                geom.select_set(True)
+                bpy.context.view_layer.objects.active = geom
+                bpy.ops.object.join()
+                
+                # Calculate the number of faces in the mesh
+                poly_count = len(geom.data.polygons)
+                print("Polygon count: ", poly_count)
+                
+                # Get the adaptive decimation ratio
+                decimation_ratio = decimation_lookup(poly_count)
+                    
+                # Add the Decimate modifier to the selected object
+                print("Applying decimation with a ratio: ", decimation_ratio)
+                decimate_modifier = geom.modifiers.new(name='Decimate', type='DECIMATE')
+                decimate_modifier.ratio = decimation_ratio
+
+                # Exporting in stl.
+                stl_filepath = osp.join(geom_dir, f"{leaf_name}.stl")
+                if (not self.rerun) or (not osp.isfile(stl_filepath)):
+                    print("Exporting STL file...")
+                    bpy.ops.export_mesh.stl(
+                        filepath=stl_filepath,
+                        check_existing=False,
+                        use_selection=True,
+                        use_mesh_modifiers=True,
+                        batch_mode='OFF',
+                        global_scale=1.0,
+                        axis_forward='Y',
+                        axis_up='Z',
+                        ascii=False,
+                    )
+                bpy.ops.object.select_all(action='DESELECT')
+
+                params = copy.deepcopy(self.p)
+                params['collision_representation']['acd']['decompose_method'] = decompose_method
+                process_mesh_args.append((stl_filepath, params, self.rerun))
+
+        print("Convex solid decomposition...")
+        i = 0
+        while True:
+            start_idx = i * self.n_workers
+            end_idx   = (i+1) * self.n_workers
+            with mp.Pool(self.n_workers) as p:
+                p.map(process_mesh, process_mesh_args[start_idx : end_idx])
+            i = i + 1
+            if end_idx >= len(process_mesh_args):
+                break
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--scene_path', required=True, help='relative path to the scene folder')
```

Comment:
- Rename to `pipeline_dir` or `data_pipeline_dir`. I'm assuming that all pipeline output will go into some top-level folder.
- Rename to `num_parallel_workkers`, rename to `actors`.
- Include optional `scene_id` to run this pipeline stage for a single scene.
- Remove help strings for consistency with our other tools and because they always get out-of-date and we don't want to maintain them.